tools/tdbg/tdbg_commands.go

```diff
@@ -180,9 +180,23 @@ func newAdminWorkflowCommands(clientFactory ClientFactory, prompterFactory Promp
 					Usage:       "Fully qualified archetype name of the execution",
 					DefaultText: chasm.WorkflowArchetype,
 				},
+				&cli.StringFlag{
+					Name:     FlagVisibilityQuery,
+					Usage:    "Visibility query to select workflows",
+					Required: true,
```

Comment:
hmm why it's required? Operator can either do batch or just refresh task for one execution.